src/workspace-hack/Cargo.toml

```diff
@@ -94,7 +95,8 @@ serde_json = { version = "1.0.99", features = ["alloc", "arbitrary_precision", "
 sha2 = { version = "0.10.6" }
 similar = { version = "2.2.1", features = ["inline", "unicode"] }
 smallvec = { version = "1.10.0", default-features = false, features = ["const_generics", "serde", "union", "write"] }
-socket2 = { version = "0.4.9", default-features = false, features = ["all"] }
+socket2-9fbad63c4bcf4a8f = { package = "socket2", version = "0.4.9", default-features = false, features = ["all"] }
+socket2-d8f496e17d97b5cb = { package = "socket2", version = "0.5.3", default-features = false, features = ["all"] }
```

Comment:
Huh, what's going on here?

Reply:
oh weird -- I'm not sure - will resolve!